auth.go

```diff
@@ -3,19 +3,37 @@ package kinetic
 import (
 	"errors"
 
-	gokinesis "github.com/rewardStyle/go-kinesis"
+	"github.com/aws/aws-sdk-go/aws"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	//gokinesis "github.com/rewardStyle/go-kinesis"
 )
 
 // ErrMetaAuthentication represents an error that occurred on authentication from meta
 var ErrMetaAuthentication = errors.New("Authentication error: failed to auth from meta.  Your IAM roles are bad, or you need to specify an AccessKey and SecretKey")
 
-func authenticate(accessKey, secretKey string) (auth *gokinesis.AuthCredentials, err error) {
+// func authenticate(accessKey, secretKey string) (auth *gokinesis.AuthCredentials, err error) {
+// 	if accessKey == "" || secretKey == "" {
+// 		if auth, err = gokinesis.NewAuthFromMetadata(); err != nil {
+// 			return nil, ErrMetaAuthentication
+// 		}
+// 	} else {
+// 		auth = gokinesis.NewAuth(accessKey, secretKey, "")
+// 	}
+// 	return
+// }
```

Comment:
can you please remove the comment code or provide another comment as to why it's commented?